Assets/MixedRealityToolkit-SDK/Features/SpatialAwareness/System/MixedRealitySpatialAwarenessSystem.cs

```diff
@@ -177,9 +177,6 @@ public override void Destroy()
                     }
                     surfaceParent = null;
                 }
-
-                // Tell the observer to clean up
-                SpatialAwarenessObserver?.Destroy();
```

Comment:
Presumably this is not required since the core toolkit component will call Destroy on the observer, correct?

Reply:
That is correct.